subset/brute/client.py

```diff
@@ -0,0 +1,73 @@
+import socket
+import sys
+import json
+
+userData=open('userList.json')
+userList = json.load(userData)
+userData.close()
+
+username= userList['username'].split(",")
+password= userList['password'].split(",")
+
+arguments = sys.argv
+
+print 'pytelnClient test begin'
+print 'address:' + arguments[1]
+print 'port:' + arguments[2]
+print 'reportFile:' + arguments[3]
+
+#address = '127.0.0.1'
+address = str(arguments[1])
+#port = 10000
+port = int(arguments[2])
+reportFile = str(arguments[3])
+dataLength =64
+#username=['root','admin','manager']
+#password=['root','pass','friend']
+expected=['login:','Password:','Last login:','Login incorrect','Connection closed by foreign host.']
+expected1=['username:','password:','success','login failed','Connection closed by foreign host.']
+
+sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+server_address = (address,port)
+
+sock.connect(server_address)
+
+loginReport='\n'
+u=0
+p=0
+
+try:
+	while True:
```

Comment:
use 4-indent not 8-indent.

Reply:
changed tabs to four spaces as suggested